contracts/VToken.sol

```diff
@@ -281,23 +276,23 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
 
     /**
      * @notice Returns the current per-block borrow interest rate for this vToken
-     * @return The borrow interest rate per block, scaled by 1e18
+     * @return rate The borrow interest rate per block, scaled by 1e18
```

Comment:
Since this isn't a named return should we name the return in the natspec?

Reply:
This is the case in a number of places

Reply:
Does it render unnamed return arguments correctly? My IDE highlights the first word after `@return` as a return value name, but maybe it's an IDE bug and it's fine to omit the name here 🤷

Reply:
I think it is an IDE bug from the generated docs I've seen it checks the functions signature before assuming the first word is a named return arg.

The IDE must not be doing that